code/modules/clothing/glasses/engine_goggles.dm

```diff
@@ -121,7 +143,16 @@
 
 	modes = list(MODE_NONE = MODE_TRAY, MODE_TRAY = MODE_NONE)
 
+/obj/item/clothing/glasses/meson/engine/shuttle
+	name = "shuttle region scanner"
+	icon_state = "trayson-shuttle"
+	item_state = "trayson-shuttle"
+	desc = "Used to see the boundaries of shuttle regions."
```

Comment:
Very nice, this will help with things like repairing the caravanambush derelict freighter.